Classes/Console/Command/ConfigurationCommandController.php

```diff
@@ -1,199 +0,0 @@
-                $this->outputLine('<info>Removed "%s" from system configuration.</info>', [$path]);
-            } else {
-                $this->outputLine('<warning>Path "%s" seems invalid or empty. Nothing done!</warning>', [$path]);
-            }
-        }
-    }
-
-    /**
-     * Show configuration value
-     *
-     * Shows system configuration value by path.
-     * If the currently active configuration differs from the value in LocalConfiguration.php
-     * the difference between these values is shown.
-     *
-     * <b>Example:</b> <code>%command.full_name% DB</code>
-     *
-     * @param string $path Path to system configuration option
-     */
-    public function showCommand($path)
-    {
-        $hasActive = $this->configurationService->hasActive($path);
-        $hasLocal = $this->configurationService->hasLocal($path);
-        if (!$hasActive && !$hasLocal) {
-            $this->outputLine('<error>No configuration found for path "%s"</error>', [$path]);
-            $this->quit(1);
-        }
-        $active = null;
-        if ($hasActive) {
-            $active = $this->configurationService->getActive($path);
-        }
-        if ($hasActive && $this->configurationService->localIsActive($path)) {
-            $this->outputLine($this->consoleRenderer->render($active));
-        } else {
-            $local = null;
-            if ($hasLocal) {
-                $local = $this->configurationService->getLocal($path);
-            }
-            $this->outputLine($this->consoleRenderer->renderDiff($local, $active));
-        }
-    }
-
-    /**
-     * Show active configuration value
-     *
-     * Shows active system configuration by path.
-     * Shows the configuration value that is currently effective, no matter where and how it is set.
-     *
-     * <b>Example:</b> <code>%command.full_name% DB --json</code>
-     *
-     * @param string $path Path to system configuration
-     * @param bool $json If set, the configuration is shown as JSON
-     */
-    public function showActiveCommand($path, $json = false)
-    {
-        if (!$this->configurationService->hasActive($path)) {
-            $this->outputLine('<error>No configuration found for path "%s"</error>', [$path]);
-            $this->quit(1);
-        }
-        $active = $this->configurationService->getActive($path);
-        $this->outputLine($this->consoleRenderer->render($active, $json));
-    }
-
-    /**
-     * Show local configuration value
-     *
-     * Shows local configuration option value by path.
-     * Shows the value which is stored in LocalConfiguration.php.
-     * Note that this value could be overridden. Use <code>typo3cms configuration:show <path></code> to see if this is the case.
-     *
-     * <b>Example:</b> <code>%command.full_name% DB</code>
-     *
-     * @param string $path Path to local system configuration
-     * @param bool $json If set, the configuration is shown as JSON
-     * @see typo3_console:configuration:show
```

Comment:
Same issue as Benni already mentioned in #836  here. How to proceed?

Reply:
for now, I'd say we just drop this feature and add this info as plain text to the help of the command. the reference will need an update then (`./typo3cms commandreference:render`)

Reply:
Done, please have a look